sensor/common/compliance/node_inventory_handler_impl.go

```diff
@@ -19,6 +24,15 @@ var (
 	errInventoryInputChanClosed = errors.New("channel receiving node inventories is closed")
 	errIndexInputChanClosed     = errors.New("channel receiving node indexes is closed")
 	errStartMoreThanOnce        = errors.New("unable to start the component more than once")
+	rhcosOSImageRegexp          = regexp.MustCompile(`(Red Hat Enterprise Linux) (CoreOS) ([0-9.-]+)`)
```

Comment:
Should we move this to `node_matcher.go`, as it is only used there?

Reply:
Addressed in [d339131](https://github.com/stackrox/stackrox/pull/13686/commits/d3391318b504b10c13b7b5da0a5de4649301b2d7)